stable/postgresql/templates/_helpers.tpl

```diff
@@ -3,26 +3,30 @@
 Expand the name of the chart.
 */}}
 {{- define "postgresql.name" -}}
-{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 24 -}}
```

Comment:
leave it as it was in the previous version. trunc at 63 characters and add the trimSuffix

```
{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
```